test/unittests/eof_validation_test.cpp

```diff
@@ -772,35 +764,31 @@ TEST_F(eof_validation, multiple_code_sections_headers)
 
 TEST_F(eof_validation, many_code_sections_1023)
 {
-    std::string code_sections_1023;
-    for (auto i = 0; i < 1022; ++i)
-        code_sections_1023 += "E5" + hex(big_endian(static_cast<uint16_t>(i + 1)));
-    code_sections_1023 += "5B5B00";
-
-    const auto code = "EF0001 010FFC 0203FF " + 1023 * bytecode{"0003"} + "040000 00" +
-                      1023 * bytecode{"00800000"} + code_sections_1023;
+    auto code = eof_bytecode(jumpf(1));
+    for (auto i = 1; i < 1022; ++i)
+        code = code.code(jumpf(static_cast<uint16_t>(i + 1)), 0, 0x80, 0);
+    code = code.code(OP_STOP, 0, 0x80, 0);
 
     add_test_case(code, EOFValidationError::success);
 }
 
 TEST_F(eof_validation, many_code_sections_1024)
 {
-    std::string code_sections_1024;
-    for (auto i = 0; i < 1023; ++i)
-        code_sections_1024 += "E5" + hex(big_endian(static_cast<uint16_t>(i + 1)));
-    code_sections_1024 += "5B5B00";
-
-    const auto code = "EF0001 011000 020400 " + 1024 * bytecode{"0003"} + "040000 00" +
-                      1024 * bytecode{"00800000"} + code_sections_1024;
+    auto code = eof_bytecode(jumpf(1));
+    for (auto i = 1; i < 1023; ++i)
+        code = code.code(jumpf(static_cast<uint16_t>(i + 1)), 0, 0x80, 0);
+    code = code.code(OP_STOP, 0, 0x80, 0);
 
     add_test_case(code, EOFValidationError::success);
 }
 
 TEST_F(eof_validation, too_many_code_sections)
 {
-    add_test_case("ef0001 011004 020401" + 1025 * bytecode{"0001"} + "040000 00" +
-                      1025 * bytecode{"00800000"} + bytecode{bytes(1025, OP_STOP)},
-        EOFValidationError::too_many_code_sections);
+    auto code = eof_bytecode(OP_STOP);
+    for (auto i = 0; i < 1024; ++i)
+        code = code.code(OP_STOP, 0, 0x80, 0);
```

Comment:
I would change this also to using jumpf to make it invalid for one reason.
```suggestion
    auto code = eof_bytecode(jumpf(1));
    for (auto i = 1; i < 1024; ++i)
        code = code.code(jumpf(static_cast<uint16_t>(i + 1)), 0, 0x80, 0);
    code = code.code(OP_STOP, 0, 0x80, 0);
```